scripts/vm/restart-es-service.sh

```diff
@@ -3,7 +3,8 @@
 shopt -s nocasematch
 
 REMOTE_USER="elkadmin"
-PRIVATE_KEY="~/.ssh/elk_private_key"
+ENVIRONMENT=$1
+PRIVATE_KEY=$2
```

Comment:
Should be locked down with SSO so less of a concern but does this show the private key in the log?

Reply:
It shouldn't no, it's just passing the path to the key rather than the contents